Stratis.Bitcoin/Features/BlockStore/LoopSteps/BlockStoreInnerStepReadBlocks.cs

```diff
@@ -3,13 +3,13 @@
 using Stratis.Bitcoin.BlockPulling;
 using System;
 using System.Collections.Generic;
-using System.Linq; 
+using System.Linq;
 using System.Threading.Tasks;
 
 namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
 {
     /// <summary>
-    /// Downloads blocks from the BlockPuller removes block from the DownloadStack.
+    /// Reads blocks from the BlockPuller and removes block from the DownloadStack.
```

Comment:
`<see>` for DownloadStack and BlockPuller 

OR

Reads blocks from the puller and removes block from the download stack.

both variants are OK towards renaming of those classes, but the original is not